sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -230,6 +230,12 @@ def test_predict_time_index(
             _assert_correct_pred_time_index(y_pred.index, cutoff, fh=fh_int)
             _assert_correct_columns(y_pred, y_train)
 
+            if estimator_instance.get_tag("capability:pred_int"):
+                y_pred_int = estimator_instance.predict_interval()
+                _assert_correct_pred_time_index(y_pred_int.index, cutoff, fh=fh_int)
+                y_pred_q = estimator_instance.predict_quantiles()
+                _assert_correct_pred_time_index(y_pred_q.index, cutoff, fh=fh_int)
+
```

Comment:
Consider addition of `predit_proba`?

Reply:
hm, yes. I thought the same but hesitated doing this here due to the current `tensorflow` dependency of the method.
Maybe splitting the blocks smartly and try/except would be an option.

Reply:
the same is actually tested in `test_predict_proba` already.